float64 q1
float64 q2
float64 q3
float64 dx
float64 dy
float64 dz
---
float64[] joint_vel
